webapp/channels/src/components/save_button.tsx

```diff
@@ -47,7 +47,7 @@ export default class SaveButton extends React.PureComponent<Props> {
             ...props
         } = this.props;
 
-        let className = 'save-button btn';
+        let className = 'btn btn-primary';
```

Comment:
Looking at where this is used, having `btn-primary` as the default for `btnClass` instead of here meant that this worked a bit differently in a couple ways:
1. `btn-primary` was removed when this button is disabled, and now it isn't. Is that change going to cause any issues?
2. People using this component can no longer remove the `btn-primary` class by adding a `btnClass` prop. That's done in a few places, but the easiest place I can find it is for the next button in the list of custom emojis in the settings pages for that. For that, you'll end up with a button with both `btn-primary` and `btn-tertiary` applied. Is that a problem, and do we need more changes to those?

Reply:
I'm also still seeing a few places in the CSS where we're using the `save-button` class. Should the rest of those be removed as well? There's only a 5 or 6 places left

Reply:
1. I think that's fine, we have a disabled style on the btn-primary right now.
2. I'll have a look at those other cases.